cassiopeia/core/league.py

```diff
@@ -156,15 +162,24 @@ def platform(self) -> Platform:
 
     @property
     def league_id(self) -> str:
-        return self._data[LeagueEntryData].leagueId
+        try:
```

Comment:
To be consistent with current code, I'd have no problems adding a decorator like the one in match
@load_match_on_attributeerror
instead of repeating these try except blocks

Reply:
This functionality is more straight forward than how @load_match_on_attributeerror is used, so this minor code duplicate will be fine if we end up with it. But see below comment.